tracking/inc/track.h

```diff
@@ -0,0 +1,20 @@
+#ifndef TRACKH
+#define TRACKH
+
+enum TrackStatus {alive, dead};
+
+struct track {
+    int index{0};
+    int pdg{0};
+    double energy{10};
+    double pos[3]{0};
+    double dir[3]{1};
+    int mother_index{0};
+    TrackStatus status{alive}; 
+    int current_process{0};
+    float interaction_length{FLT_MAX};
+    float energy_loss{0}; // primitive version of scoring 
+    int number_of_secondaries{0}; // primitive version of scoring
+  };
+
+#endif
```

Comment:
Some "no newline at end of file" indicate that clang-format may not be applied.

Reply:
Yes, some other whitespace also doesn't look correctly formatted. For `.cu` files, you have to check if your `clang-format` is recent enough to recognize this extensions. My patch for that is a few years old already, but not sure if it already propagated to all distributions...

Reply:
Looks like the system does not check it any more. @drbenmorgan looks clang formatting  was removed as action, right?